src/main/scala/nl/knaw/dans/easy/stage/fileitem/EasyFilesAndFolders.scala

```diff
@@ -16,27 +16,28 @@
 package nl.knaw.dans.easy.stage.fileitem
 
 import java.io.File
-import java.sql.{DriverManager, PreparedStatement}
+import java.sql.{ Connection, DriverManager, PreparedStatement }
 
+import nl.knaw.dans.easy.stage.ExistingAncestor
 import nl.knaw.dans.easy.stage.lib.Props.props
 
 import scala.annotation.tailrec
 import scala.util.Try
 
 trait EasyFilesAndFolders {
-  def getExistingAncestor(file: File, datasetId: String): Try[(String,String)]
+  def getExistingAncestor(file: File, datasetId: String): Try[ExistingAncestor]
```

Comment:
I like the use of this type alias, but why not put its declaration in the `fileitem` package? That's the only place it is used...